src/Darryldecode/Cart/Cart.php

```diff
@@ -784,4 +755,14 @@ public function setThousandsSep($thousands_sep)
     {
         $this->thousands_sep = $thousands_sep;
     }
+
+    /**
+     * @param $name
+     * @param $value
+     * @return mixed
+     */
+    protected function fireEvent($name, $value = [])
+    {
+        return $this->events->fire($this->getInstanceName() . '.' . $name, array_values([$value, $this]), true);
```

Comment:
Event halting is to prevent executing further events if one returns false. Otherwise we cannot stop the events from the code if we want to.
During the triggered event you can return false to stop further events or not return anything. If you don't return anything that will continue with all the events.

But I don't use this change anymore, so what do you think is more consistent for the whole cart.